test/e2e/page-objects/pages/dialog/account-details-modal.ts

```diff
@@ -1,10 +1,19 @@
+import { tEn } from '../../../../lib/i18n-helpers';
+import { LavaDomeDebug } from '@lavamoat/lavadome-core';
 import { Driver } from '../../../webdriver/driver';
+import { WALLET_PASSWORD } from '../../../helpers';
```

Comment:
unrelated to the PR, but in case you want to fix it: I would add this variable in the`e2e/constants.ts` file and then import it from there here.
There are a couple of specs that use that, so I would gradually be updating them to use the variable in the constants file instead of the helpers one, and eventually remove this variable from helpers (together with the method that uses that)

What do you think?